build.sbt

```diff
@@ -10,8 +10,8 @@ def localSnapshotVersion: String = s"$parseTagVersion-SNAPSHOT"
 def isCI = System.getenv("CI") != null
 
 def scala211 = "2.11.12"
-def scala212 = "2.12.8"
-def scala213 = "2.13.1"
+def scala212 = "2.12.12"
+def scala213 = "2.13.3"
```

Comment:
this says that the warning is a linter problem in this specific version and can be disabled:

https://contributors.scala-lang.org/t/multiarg-infix-application-considered-warty/4490

do you really believe that scalatest matchers will cease to exist completely?

Reply:
Idiomatic usage of ScalaTest matchers heavily relies on nested infix operators, which are deprecated to my best understanding. It's maybe possible to tweak the build settings to suppress these deprecation warnings so we can continue to use matchers. However, matchers are only used in one test suite so I figured it was easier to remove them altogether.

Personally, I think matchers are difficult to read and write so I think it's nice to get rid of them.